jina/drivers/rank/__init__.py

```diff
@@ -82,29 +76,36 @@ def _apply_all(self, docs: 'DocumentSet', *args, **kwargs) -> None:
             )
 
             matches = doc.matches
-            old_match_scores = {match.id: match.score.value for match in matches}
-            match_meta = (
-                {match.id: match.get_attrs(*self._exec_match_keys) for match in matches}
-                if self._exec_match_keys
-                else None
-            )
+            num_matches = len(matches)
+            old_match_scores = []
+            extract_match_meta = self._exec_match_keys is not None
```

Comment:
can we rename this variable? `extract_match_meta` sounds like a function name, while it's actually a boolean variable.

Reply:
can u suggest a name inline?